.github/workflows/ci-openapi.yml

```diff
@@ -137,3 +137,54 @@ jobs:
             <!--openapi-diff-workflow-comment-->
 
             No changes to OpenAPI specification found. See history of this comment for previous changes.
+
+  publish:
+    name: OpenAPI - Publish Unstable Spec
+    if:
+      - github.event_name != 'pull_request'
+      - contains(github.repository_owner, 'jellyfin')
+    runs-on: ubuntu-latest
+    needs:
+      - openapi-head
+    steps:
+      - name: Set unstable dated version
+        id: version
+        run: |-
+          echo "JELLYFIN_VERSION=$(date +'%Y%m%d%H')" >> $GITHUB_ENV
```

Comment:
Won't this overwrite the existing spec if there are two merges on the same day?

Reply:
Within one hour, yes. I could make it granular to the second if we wish, I was trying to keep versioning similar to the current unstables but this might cause a problem yes.